integration-tests/swiss-knife/project/cloudflow-plugins.sbt

```diff
@@ -0,0 +1,7 @@
+// Resolver for the cloudflow-sbt plugin
+//
+resolvers += "Akka Snapshots" at "https://repo.akka.io/snapshots/"
+resolvers += Resolver.url("cloudflow", url("https://lightbend.bintray.com/cloudflow"))(Resolver.ivyStylePatterns)
+
+addSbtPlugin("com.lightbend.cloudflow" % "sbt-cloudflow" % "1.3.2-SNAPSHOT")
```

Comment:
We will need to make sure this gets updated as part of the release automation tooling. This happens automatically for (known) examples but this one breaks that pattern and will have to be added manually.